src/components/wrapper/wrapper.js

```diff
@@ -1,43 +1,68 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { connect } from 'react-redux';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { useApolloQuery } from '../../apollo/utils';
+import { client } from '../../apollo/config';
+import { GraphQLProvider } from '../provider/provider';
+import { GET_VERSIONS } from '../../apollo/queries';
 import { isLoading } from '../../selectors/loading';
 import classnames from 'classnames';
 import GlobalToolbar from '../global-toolbar';
 import FlowChartWrapper from '../flowchart-wrapper';
 import ExperimentWrapper from '../experiment-wrapper';
 import SettingsModal from '../settings-modal';
-import { Provider } from '../provider/provider';
+import UpdateReminder from '../update-reminder';
 
 import './wrapper.css';
 
 /**
  * Main app container. Handles showing/hiding the sidebar nav, and theme classes.
  */
-export const Wrapper = ({ theme }) => (
-  <Provider useMocks={false}>
-    <div
-      className={classnames('kedro-pipeline kedro', {
-        'kui-theme--dark': theme === 'dark',
-        'kui-theme--light': theme === 'light',
-      })}
-    >
-      <h1 className="pipeline-title">Kedro-Viz</h1>
-      <Router>
-        <GlobalToolbar />
-        <SettingsModal />
-        <Switch>
-          <Route exact path={['/', '/flowchart']}>
-            <FlowChartWrapper />
-          </Route>
-          <Route path={['/runsList', '/runsList/:id']}>
-            <ExperimentWrapper />
-          </Route>
-        </Switch>
-      </Router>
-    </div>
-  </Provider>
-);
+export const Wrapper = ({ theme }) => {
+  const { data } = useApolloQuery(GET_VERSIONS, { client });
```

Comment:
Should we reassign `data` to something more specific, like `vizVersionData`? As this is such a top-level component it could be better to be more explicit about what this is here.

Reply:
Thanks for the suggestion, I understand your point with the confusion, but it is not technically feasible to be amended here given the data is wrapped exactly under the `data` field in the graphql response and I would need to directly reference that in the JS object destructing here. 

Moreover, the `data` field is not used directly in the component here but rather used to update the values of the internal hooks, so there isn't any need to reassign it to another variable as that variable won't be used / redundant.

Reply:
I believe you meant assigning the data to an alias - I've just done that in my latest commit

Reply:
Yeah exactly, rename while destructuring.